package/rebuild-todo

```diff
@@ -219,6 +224,22 @@ read <&1
 
 pkgctl repo clone "${packages[@]}"
 
+if ((IMPORT_KEYS)); then
+	echo "Importing PGP keys..."
+	# Only add paths that actually have key(s) to import and ignore paths that don't, don't exit on error
+	key_paths=($(find "${packages[@]/%//keys/pgp}" -type f 2>/dev/null || true))
+
+	if [[ "${#key_paths[@]}" -ne 0 ]]; then
+		if [[ -z "$OFFLOAD" ]]; then
+			cat "${key_paths[@]}" | gpg --import
+		else
+			cat "${key_paths[@]}" | ssh build.archlinux.org gpg --import
+		fi
```

Comment:
```suggestion
		cat "${key_path[@]}" | if [[ -z "$OFFLOAD" ]]; then
			gpg --import
		else
			ssh build.archlinux.org gpg --import
		fi
```

Then you only need to change the cat once if we need to do more things later.

Reply:
smart